update-hosts.sh

```diff
@@ -9,7 +14,7 @@ if grep "FUN host entries" /etc/hosts >/dev/null; then
   exit 0
 fi
 
-sudo sed -i '/ksqldb-server/d' /etc/hosts
+sudo sed -i '' '/ksqldb-server/d' /etc/hosts
```

Comment:
This fixes an error that was occurring when running the script on macOS zsh shell.